lua/lvim/plugins.lua

```diff
@@ -272,4 +273,10 @@ return {
     end,
     disable = not lvim.builtin.terminal.active,
   },
+
+  -- SchemaStore
+  {
+    "b0o/schemastore.nvim",
+    commit = commit.schemastore,
```

Comment:
should this be loaded on `ft = 'json'`? or maybe it's not that heavy anyway?

Reply:
sadly lazy loading causes a lot of weird issues on this one